### PR TITLE
DOCS-7714 cms threats active protection

### PR DESCRIPTION
**Pull request by michaelcretzman**
DOCS-7714

### What does this PR do? What is the motivation?

- tells the user how to enable and use the CMS Threats Active Protection feature

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes
- Needs editorial review: DOCS-7858

### CODE REVIEW

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+<div class="alert alert-info">Blocking is applied to all threats detected after blocking is enabled. Blocking is not retroactive.</div>
+
+## Active Protection availability
+
+Active Protection is enabled at the organization level. 
+
+<div class="alert alert-info">Active Protection blocking functionality is available in a subset of the OOTB Agent rules only. Agent rule monitoring runs regardless of whether Active Protection is enabled.</div>
+
+To see if Active Protection is already enabled in your organization, go to [Agent Configuration][2] and see if there is a **Protection** column in the Agent rule list.
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:41]:
```suggestion
To check if Active Protection is already enabled in your organization, go to [Agent Configuration][2]. If Active Protection is enabled, a **Protection** column is displayed in the Agent rule list.
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+### Scoping the Agent rule
+
+When you create or edit an Agent rule after Active Protection is enabled, you can select **Blocking** in the rule **Protection** setting. 
+
+When you select **Blocking**, you can scope where Datadog should apply the rule using the **Everywhere** and **Custom** options. 
+
+#### Everywhere
+
+The rule applies to all services, hosts, images, etc.
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:107]:
```suggestion
The rule applies to all services, hosts, and images.
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+#### Custom
+
+In **Custom**, you can specify services or tags to automatically generate an expression for where to apply blocking protection.
+
+<div class="alert alert-info">Any service or image that is not matched by the expression is not blocked, but it is still monitored.</div>
+
+You can use services and tags to generate an expression. Datadog matches the rule using the service(s) or tag(s) you provide.
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:115]:
```suggestion
You can use services and tags to generate an expression. Datadog matches the rule using the services or tags you provide.
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+- **Services:** Enter one or more service names. You can use wildcards. For example, entering `a*` generates the expression `process.envp in ["DD_SERVICE=a*"]`.
+- **Tags:** Enter one or more tags for container images. If you enter multiple tags, all tags must match for the **Protection** to apply. There are two options:
+  - `image_tag`: the image tag only. For example, `stable-perl`.
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:119]:
```suggestion
  - `image_tag`: The image tag only. For example, `stable-perl`.
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+  - `short_image`: the image name without a tag. For example, `nginx`.
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:120]:
```suggestion
  - `short_image`: The image name without a tag. For example, `nginx`.
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  - For example, a Github Container registry image such `ghcr.io/MY_NAMESPACE/MY_IMAGE:2.5` can be referenced using:
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:121]:
```suggestion
  - For example, a Github Container registry image such as `ghcr.io/MY_NAMESPACE/MY_IMAGE:2.5` can be referenced using:
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    - `image_tag`: `2.5`.
+    - `short_image`: `MY_IMAGE`.
+
+## Blocked attack example
+
+After Active Protection is enabled at set to **Blocking** for an Agent rule, blocked threats appear in [Signals][1].
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:127]:
```suggestion
After Active Protection is enabled and set to **Blocking** for an Agent rule, blocked threats appear in [Signals][1].
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+A signal for a blocked threat contains the messages `SECURITY RESPONSE` and `The malicious process THREAT NAME has automatically been killed.`:
```
Comment (drichards-87) [inline content/en/security/cloud_security_management/guide/active-protection.md:129]:
```suggestion
A signal for a blocked threat contains the messages `SECURITY RESPONSE` and `The malicious process <THREAT NAME> has automatically been killed.`:
```

**Review by drichards-87 — APPROVED** [review]
Left some light feedback and approved the PR.

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+By default, all OOTB Agent rules are in a monitoring state. Enabling Active Protection does not immediately change the default state. Enabling Active Protection allows you to change the state of a rule from monitoring to blocking.
+
+Consequently, you do not need to worry that enabling Active Protection immediately changes the state of threat detection.
+
+To enable Active Protection:
+
+1. Go to CSM [Agent Configuration][2] rules.
+2. Select **Enable Active Protection**.
+
+    {{< img src="security/cws/guide/enable-active-protection.png" alt="Your image description" style="width:100%;" >}}
```
Comment (michaelcretzman) [inline content/en/security/cloud_security_management/guide/active-protection.md:74]:
```suggestion
    {{< img src="security/cws/guide/enable-active-protection.png" alt="Enable Active Protection button" style="width:100%;" >}}
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+    - `image_tag`: `2.5`.
+    - `short_image`: `MY_IMAGE`.
+
+## Blocked attack example
+
+After Active Protection is enabled at set to **Blocking** for an Agent rule, blocked threats appear in [Signals][1].
+
+A signal for a blocked threat contains the messages `SECURITY RESPONSE` and `The malicious process THREAT NAME has automatically been killed.`:
+
+{{< img src="security/cws/guide/active-protection-signal-messages.png" alt="Your image description" style="width:100%;" >}}
```
Comment (michaelcretzman) [inline content/en/security/cloud_security_management/guide/active-protection.md:131]:
```suggestion
{{< img src="security/cws/guide/active-protection-signal-messages.png" alt="Signal messages" style="width:100%;" >}}
```

File: content/en/security/cloud_security_management/guide/active-protection.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
+---
+title: Proactively block threats with Active Protection
+kind: documentation
+further_reading:
+- link: "security/threats/workload_security_rules"
+  tag: "Documentation"
+  text: "CSM Threats Detection Rules"
+---
+
+<div class="alert alert-info">CSM Threats **Active Protection** is in beta.</div>
```
Comment (michaelcretzman) [inline content/en/security/cloud_security_management/guide/active-protection.md:10]:
```suggestion
<div class="alert alert-info">CSM Threats Active Protection is in beta.</div>
```

**Pull request merged**